promptsource/templates/mc_taco/templates.yaml

```
@@ -97,11 +99,103 @@ templates:
     metadata: !TemplateMetadata
       _do_eval: false
       _do_train: true
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
       original_task: false
-    name: mc_taco_5
+    name: generate_answer_from_question_and_context
     reference: Generate answer from question+context (if plausible)
+  8423a3fa-adcf-4d36-b639-774bd13ac3fe: !Template
+    answer_choices:
+    - 'No'
+    - 'Yes'
+    answer_choices_key: null
+    id: 8423a3fa-adcf-4d36-b639-774bd13ac3fe
+    jinja: 'Here''s what happened: {{sentence}}
+
+
+      I asked my friend {{question}}
+
+
+      and they said {{answer}}
+
+
+      Should I believe them?
+
+      |||
+
+      {{answer_choices[label]}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      original_task: true
+    name: asked_my_friend
+    reference: ''
+  89aaa7f3-d409-4a27-acd5-a207b431b12c: !Template
+    answer_choices:
+    - 'no'
+    - 'yes'
```

for consistency (doesn't matter much since we are doing rank classification):
```suggestion
    - 'No'
    - 'Yes'
```

---

Consistency is good! Fixed. 😄